.env.sample

```diff
@@ -0,0 +1,2 @@
+NODE_ENV=development
```

Comment:
dont commit env files. Keep them locally

Reply:
this is sample file, actual is not committed